NEWS

```diff
@@ -1,7 +1,63 @@
 Geany 2.0 (unreleased)
+
+    General
+    * Split "session data" into session.conf, preferences are written to
+      and read from geany.conf (PR#2776, PR#2779, PR#3014, PR#3125 and more).
+    * Add new build system Meson (experimental) (PR#2761).
+    * Require GTK 3.24 (#3580).
+
+    Bug fixes
+    * (Re-)Add ability to scroll over document tabs (Luke Horwell, PR#3134).
+    * Fix keyword colorization on filetype change (PR#3553).
+
     Interface
     * The document list in the sidebar has a new tree view. This mode is the
       new default and existing installations automatically use it (PR#1813).
+    * Use dark-theme friendly colors for compiler messages (PR#3013).
+    * Add a confirmation dialog on search & replace for the whole session
+      (PR#3033).
+    * Filter entry for symbol tree (PR#3055).
+    * Simplify project creation from existing directories with sources
+      (PR#3042).
+    * Add option to show symbols in symbol tree without category groups
+      (PR#3172).
+    * Add option to only show line endings if they differ from file default
+      (PR#3287).
+    * Make tab label length and window title length configurable
+      (Abdul Rafey, #3365).
+    * Make Go to Symbol commands show signature list (PR#3475).
+
+    Editor
+    * Update Scintilla to 5.3.7 and Lexilla to 5.2.7 (PR#2867, PR#3551).
+    * Add "Change history" feature (PR#3551).
```

Comment:
Might mention that it's disabled by default (without explaining why)

Reply:
`    * Add "Change history" feature, disabled by default (PR#3551).`

I don't know if it's worth explaining why it is disabled. The explaination will probably be longer than a NEWS item should be.

Reply:
That's why I said "(without explaining why)" ;-)

Reply:
Oops, my tired brain read "with explaining why". Sorry.

Reply:
Done.